Makefile

```
@@ -0,0 +1,157 @@
+
+# Image URL to use all building/pushing image targets
+IMG ?= controller:latest
```

Default name could be a bit more specific?
```suggestion
IMG ?= compass-manager:latest
```

---

`compass-manager-controller`? 😛
`compass-controller`?